src/setup/VisualStudioEditorsSetup/Directory.Build.props

```diff
@@ -1,11 +1,8 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
```

Comment:
These empty Build.props files can be deleted now and the projects will automatically pick up the parent.